File: LAB1/Python/lab1_3.py
name = input("Name: ")
lab_grade = float(input("Lab: "))
midterm_grade = float(input("Midterm: "))
final_grade = float(input("Final: "))


last_score = (lab_grade * 0.25) + (midterm_grade * 0.35) + (final_grade * 0.4)

print("Name: " + name)
print("Lab: " + str(lab_grade))
print("Midterm: " + str(midterm_grade))
print("Final: " + str(final_grade))
print("Last Score: " + str(last_score))

File: LAB1/C++/lab1_3.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string name;
    double labGrade, midtermGrade, finalGrade, lastScore;

    cout << "Please enter your name: ";
    cin >> name;

    cout << "Please enter your lab grade: ";
    cin >> labGrade;

    cout << "Please enter your midterm grade: ";
    cin >> midtermGrade;

    cout << "Please enter your final grade: ";
    cin >> finalGrade;

    lastScore = labGrade * 0.25 + midtermGrade * 0.35 + finalGrade * 0.4;

    cout << "Name: " << name << endl;
    cout << "Lab: " << labGrade << endl;
    cout << "Midterm: " << midtermGrade << endl;
    cout << "Final: " << finalGrade << endl;
    cout << "Last Score: " << lastScore << endl;

    return 0;
}
